src/android/app/src/main/res/values/arrays.xml

```diff
@@ -181,6 +181,31 @@
         <item>5</item>
     </integer-array>
 
+    <string-array name="staticThemeNames">
+        <item>Blue (Default)</item>
+        <item>Cyan</item>
+        <item>Red</item>
+        <item>Green</item>
+        <item>Lime Green</item>
```

Comment:
Perhaps change to just "Lime"?

Reply:
Allright,  Also would you want it to he the default color

Reply:
Either same as it is in the current release or Material You. This is up for debate, may be worth having a back and forth in the Discord channel about it

Reply:
What is the current behaviour of the Material You option on older Android versions by the way? I would recommend it just mimicking the default layout as a fallback.

Reply:
Right now, Material you is disabled by default, and the toggle is hidden if Android 11 or earlier is 

It's not hard to enable it if Android 12 is detected. Just let me know of you want to